packages/cli/src/commands/link/link.ts

```diff
@@ -98,9 +98,8 @@ export default {
       parse: (val: string) => val.toLowerCase().split(','),
     },
     {
-      name: '--all [boolean]',
+      name: '--all',
       description: 'Link all native modules and assets',
-      parse: (val: string) => val.toLowerCase().split(','),
```

Comment:
Not sure about this one, we don't support comma separated values, nor `--all true`, so just sticking to `--all`.